app/src/main/java/com/nervousfish/nervousfish/service_locator/EntryActivity.java

```diff
@@ -21,7 +21,8 @@
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        final IServiceLocator serviceLocator = ServiceLocatorCreator.createInstance();
+        final IServiceLocator serviceLocator = ServiceLocatorCreator.createInstance(
```

Comment:
It seems odd to me that this works, but I think I'll be able to come up with a solution that is a bit nicer given this possibility 😃 Is it OK if I continue developing on this branch?